Makefile.pre.in

```diff
@@ -2729,6 +2729,10 @@ patchcheck: all
 
 .PHONY: check-limited-abi
 check-limited-abi: all
+	# Regenerate 3 files using using Tools/build/stable_abi.py:
+	# - Lib/test/test_stable_abi_ctypes.py
+	# - Modules/_testcapi_feature_macros.inc
+	# - PC/python3dll.c
```

Comment:
nice!

Reply:
This *checks* if those files are up to date. For regenerating them there's `make regen-limited-abi`, which passes `--generate-all` rather than `--all`.

Also, this list of files is incomplete and it can grow in the future. Run `./Tools/build/stable_abi.py --list .` to see the full list. (I should change it so a dummy filename argument isn't necessary...)

Perhaps just link to https://devguide.python.org/developer-workflow/c-api/#adding-a-new-definition-to-the-limited-api here?